app/src/main/res/menu/activity_view_files_actions.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <item
+        android:id="@+id/item_sort"
+        android:title="@string/sort" />
```

Comment:
Add these options settings also here:
        android:icon="@drawable/sort_icon"
        app:showAsAction="always"

Use this icon
[https://material.io/icons/#ic_sort](https://material.io/icons/#ic_sort)

This is so that it is displayed as an icon, instead of overflow menu.